packages/store/ts/library/config/storeConfig.ts

```diff
@@ -163,7 +157,7 @@ export const zTablesConfig = z.record(zTableName, zTableConfig).transform((table
   // default name depends on tableName
   for (const tableName of Object.keys(tables)) {
     const table = tables[tableName];
-    table.name ??= tableName.slice(0, STORE_SELECTOR_MAX_LENGTH);
+    table.name = tableName.slice(0, STORE_SELECTOR_MAX_LENGTH);
```

Comment:
is there anything we can do at the type hint level to make folks aware of this limitation? like should a >16 length string fail TS? or warn? does zod provide any helpers for this?